bitnami/cassandra/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: cassandra
-version: 1.1.0
+version: 2.0.0
```

Comment:
We're including instructions about how to upgrade when increasing the major version? Could you add them?

Reply:
Nice!